backend/services/academics/my_courses.py

```diff
@@ -99,4 +104,109 @@ def _to_section_overview(self, section: SectionEntity) -> SectionOverview:
         return SectionOverview(
             number=section.number,
             meeting_pattern=section.meeting_pattern,
+            oh_section_id=section.office_hours_id,
+        )
+
+    def get_course_roster(
+        self,
+        user: User,
+        term_id: str,
+        course_id: str,
+        pagination_params: PaginationParams,
+    ) -> Paginated[CourseMemberOverview]:
+        """
+        Get a paginated list of members for a course.
+
+        Returns:
+            Paginated[CourseMemberOverview]
+        """
+
+        # Start building the query
+        member_query = (
+            select(SectionMemberEntity)
+            .join(SectionEntity)
+            .join(UserEntity)
+            .where(
+                SectionEntity.term_id == term_id,
+                SectionEntity.course_id == course_id,
+            )
+            .options(joinedload(SectionMemberEntity.user))
```

Comment:
Is there an intentional reason not to do the `joinedload` of `SectionMemberEntity.section` here, in addition to `.user`? I can imagine some potential reasons, but I would personally still default to going ahead and eagerly loading it in the one query rather than having SQLAlchemy lazily load it in one or more follow-on queries. That said, if you read documentation that would suggest letting an oft-repeated related entity be loaded lazily, I would defer to it.